src/main/java/ca/warp7/frc2020/Constants.java

```diff
@@ -91,6 +91,13 @@
 
     public static final double intakingSpeed = 0.3; // percent
 
+    //Feeder Constants
+
+    public static final double feedingSpeed = 0.6; //percent
+
+    //Hopper constants
+
+    public static final double hopperSpeed = 0.6; //percent
```

Comment:
can you rename this so that it starts with k, following the convention for constants?